lazycoder1412/EvaluateDivision.cpp

```diff
@@ -0,0 +1,34 @@
+class Solution {
+public:
+    map<string,vector<pair<string,double> > >graph;
+    map<string,bool> vis;
+    vector<double> ans;
+    double dfs(string start,string end){
+        vis[start]=true;
+        if(start==end)
+            return 1.0;
+        for(auto nodes: graph[start])
+            if(vis[nodes.first]==false){
+                double temp= dfs(nodes.first,end);
```

Comment:
`temp` is usually a bad variable name. Try giving this a meaningful variable name or add a comment about what this variable is supposed to do.